tests/b-cards.test.ts

```diff
@@ -1,109 +1,60 @@
-import { Scorer } from '../src/scorer';
+import { Player, Scorer } from '../src/scorer';
 import { zeroPointColourlessCard, zeroPointGreenCard } from './helpers';
 
-describe('Using BIOENGINEERED PLAGUE + other cards', () => {
```

Comment:
this is a catastophe... can it go in a catastophe.test file.. :)

Reply:
It was moved!